test/cpp/interop/xds_interop_client.cc

```diff
@@ -154,11 +158,20 @@ class TestClient {
             std::chrono::system_clock::now() + std::chrono::seconds(INT_MAX);
       }
     }
+    SimpleRequest request;
+    if (absl::GetFlag(FLAGS_response_payload_size) > 0) {
```

Comment:
Confirmed with @yashykt. Flag parsing can be expensive, and `AsyncUnaryCall` method runs in a loop with as fast as the qps argument set.

Instead, we should parse flags early, do proper validation, and provide them to `AsyncUnaryCall` via `RpcConfig& config`. Let's add new fields `response_size` and `payload` (pointing to precreated payload body) to the `RpcConfig` struct.

Reply:
Fixed.